yarn.lock

```diff
@@ -4599,6 +4599,16 @@ babel-loader@8.0.5:
     mkdirp "^0.5.1"
     util.promisify "^1.0.0"
 
+babel-loader@^8.0.6:
+  version "8.0.6"
+  resolved "https://registry.npmjs.org/babel-loader/-/babel-loader-8.0.6.tgz#e33bdb6f362b03f4bb141a0c21ab87c501b70dfb"
```

Comment:
Can you please use `https://registry.yarnpkg.com` as a registry? When you cannot change your local yarn config, can you add a `.yarnrc` in this project [as described here](https://stackoverflow.com/a/46000761/2013738)?